### PR TITLE
Feat: create dive deeper feed

### PR DESCRIPTION
**Pull request by atomictangerine**
Adds Dive Deeper section to the main page. Expands on @clairehhelms ' guide ranking system + grids.
Based on internal Jira Ticket WPDS-1093, WPDS-1023, WPDS-1024.

Screen Shot 2022-07-12 at 5 00 18 PM
Screen Shot 2022-07-12 at 5 00 10 PM
Screen Shot 2022-07-12 at 5 00 04 PM
Screen Shot 2022-07-12 at 4 59 48 PM
Screen Shot 2022-07-12 at 4 59 43 PM
Screen Shot 2022-07-12 at 4 59 35 PM
Screen Shot 2022-07-12 at 4 59 30 PM

// Will squash to use conventional commits

### CODE REVIEW

File: components/Markdown/Components/link.js [hunk 1]
```diff
@@ -33,12 +33,10 @@ export default function CustomLink({
...
 
   return (
-    <>
-      <Link as={as} href={href} forceHref passHref>
-        <A id={id} signal={useSignal ? "showColor" : ""} {...otherProps}>
-          {children}
-        </A>
-      </Link>
-    </>
+    <Link as={as} href={href} forceHref passHref>
```
Comment (artmsilva) [inline components/Markdown/Components/link.js:36]: 💯

File: pages/index.js [hunk 2]
```diff
@@ -1,44 +1,51 @@
...
+    bottom: 0,
+    position: "absolute",
+  },
+});
+
+const Index = ({ recentPosts, rankedArticles }) => {
   return (
     <>
-      <Grid>
+      <LandingContentGrid size={"wide"}>
```
Comment (artmsilva) [inline pages/index.js]:
```suggestion
      <LandingContentGrid size="wide">
```
⛏️

File: pages/index.js [hunk 3]
```diff
@@ -193,46 +209,120 @@ export default function Index({ recentPosts }) {
...
-          </CustomLink>
-        </Box>
-      </Grid>
+            <Header
+              as="h2"
+              css={{
+                borderTop: "1px solid $subtle",
+                marginTop: theme.sizes[200],
+                paddingTop: theme.sizes[100],
+                "@sm": { paddingBottom: "0", marginBottom: 0 },
```
Comment (artmsilva) [inline pages/index.js]: ⛏️ 
```suggestion
                "@sm": { paddingBottom: 0, marginBottom: 0 },
```

File: pages/utils.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
+import { Header } from "~/components/Markdown/Components/headers";
```
Comment (artmsilva) [inline pages/utils.js]: This created a new route/page in our app. https://wpds-docs-git-feat-create-dive-deeper-feed.preview.now.washingtonpost.com/utils. You could place these utils in a new folder in /components.

File: public/sitemap.xml [hunk 5]
```diff
@@ -237,4 +237,8 @@
   <url>
     <loc>https://build.washingtonpost.com/support/supported-platforms</loc>
   </url>
+
+  <url>
+    <loc>https://build.washingtonpost.com/utils</loc>
```
Comment (artmsilva) [inline public/sitemap.xml]: See above comment. https://wpds-docs-git-feat-create-dive-deeper-feed.preview.now.washingtonpost.com/utils

**Review by artmsilva — CHANGES_REQUESTED** [review]
I left some notes. Let's look into moving the utils.js file into components directory.

File: docs/resources/workshops/developer_4-7-22.mdx [hunk 6]
```diff
@@ -1,9 +1,13 @@
 ---
 title: WPDS v1 - Developer Workshop 2
-description: Learn everything you need to know about the new WPDS
+description: Watch Arturo Silva lead our second developer workshop for our v1
+  rollout of the new WPDS. Same topisc as the first one, but with
+  a different approach.
 kicker: Workshops
 publishDate: April 7, 2022
 imageTag: /img/resources/workshops/dev2.png
+imageAltText: To the right of the image, a man stands holding a text box with both hands. In front of him a large website floats in front of him. He appears to be inserting the text box into the upper left of the website. On the upper left of the image, the word 'WORKSHOP' appears in large bold letters. The words below this say 'Session 2 - WPDS for Developers'. 'WP Design System' appears on the lower left.
```
Comment (clairehhelms) [inline docs/resources/workshops/developer_4-7-22.mdx:9]: I like the alt text!

File: pages/resources/index.js [hunk 7]
```diff
@@ -190,34 +147,26 @@ async function getProps(collections, recents) {
...
       size = THUMBNAIL_SQUARE;
     } else {
-      posts = [...item.docs].slice(0, 3);
+      posts = [...collection.docs].slice(0, 3);
       size = THUMBNAIL_WIDE;
     }
-    let category = { name, posts, description, size, type, id };
-    return category;
+    //return category
+    return { name, posts, description, size, type, id };
```
Comment (clairehhelms) [inline pages/resources/index.js:178]: thank you for cleaning these up!! <3

**Review by clairehhelms — APPROVED** [review]

**Review by artmsilva — APPROVED** [review]

**Pull request merged**